frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -72,24 +71,30 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
     const displayOnly = !!display;
     const cloneBaseDiskImage = iGetFieldValue(cloneBaseDiskImageField);
     const mountWindowsGuestTools = iGetFieldValue(mountWindowsGuestToolsField);
+    const isUserTemplateLoaded = iGetIsLoaded(userTemplate);
+    const userTemplateName: string = isUserTemplateLoaded
+      ? iGetName(iGetLoadedData(userTemplate))
+      : '';
     const params = {
-      userTemplate,
+      userTemplateName,
       flavor,
       workload: workloadProfile,
       os,
     };
 
-    const vanillaTemplates = immutableListToShallowJS(
-      concatImmutableLists(iGetLoadedData(commonTemplates), iGetLoadedData(userTemplates)),
-    );
+    const vanillaTemplates = isUserTemplateLoaded
+      ? immutableListToShallowJS(
```

Comment:
let's just use the the usertemplate here when selected and not mash them up with commontemplates